numba_dpex/experimental/_kernel_dpcpp_spirv_overloads/_private_array_overloads.py

```diff
@@ -23,55 +24,12 @@
 )
 from numba_dpex.utils import address_space as AddressSpace
 
-from ..target import DPEX_KERNEL_EXP_TARGET_NAME
+from ._registy import lower
```

Comment:
Typo in module name.
```suggestion
from ._registry import lower
```